include/fc/interprocess/container.hpp

```diff
@@ -139,5 +135,28 @@ namespace fc {
          for( auto& item : value )
              fc::raw::unpack( s, item );
        }
+
+       template<typename Stream, typename T, typename... A>
+       inline void pack( Stream& s, const bip::set<T,A...>& value ) {
+         pack( s, unsigned_int((uint32_t)value.size()) );
+         auto itr = value.begin();
+         auto end = value.end();
+         while( itr != end ) {
+           fc::raw::pack( s, *itr );
+           ++itr;
+         }
+       }
+       template<typename Stream, typename T, typename... A>
+       inline void unpack( Stream& s, bip::set<T,A...>& value ) {
+         unsigned_int size;
+         unpack( s, size );
+         value.clear();
+         for( uint32_t i = 0; i < size.value; ++i )
+         {
+             T tmp;
+             fc::raw::unpack( s, tmp );
+             value.insert( std::move(tmp) );
```

Comment:
I think you should remove `std::move` as tmp needs to be copied into `bip::set` using the allocator of `bip::set`